react/components/SearchBar/components/AutocompleteInput.js

```diff
@@ -172,8 +172,20 @@ const AutocompleteInput = ({
   })
 
   return (
+    <form action="#" onSubmit={e => {
+      e.preventDefault()
+      e.stopPropagation()
+      e.nativeEvent.stopImmediatePropagation()
```

Comment:
Add comments to why these are being used.

Reply:
For example, if I search 'batman' when you click Enter on the keyboard, the URL changes with the searched word like this: /batman?_q=batman&map=ft and immediatly after execute the form action '#' and change the URL /#?_q=&map=ft.
So, we need to capture the event onSubmit to avoid to go to the form action "#"